src/Advection/weno_interpolants.jl

```diff
@@ -170,107 +171,86 @@ end
+        @inline zweno_alpha_loop(scheme::WENO{$buffer, FT}, β, τ)   where FT = @inbounds $(metaprogrammed_zweno_alpha_loop(buffer))
     end
 end
 
-# Global smoothness indicator τ₂ᵣ₋₁ taken from "Accuracy of the weighted essentially non-oscillatory conservative finite difference schemes", Don & Borges, 2013
+# Global smoothness indicator τ₂ᵣ₋₁ from "Accuracy of the weighted essentially non-oscillatory conservative finite difference schemes", Don & Borges, 2013
 @inline global_smoothness_indicator(::Val{2}, β) = @inbounds abs(β[1] - β[2])
 @inline global_smoothness_indicator(::Val{3}, β) = @inbounds abs(β[1] - β[3])
 @inline global_smoothness_indicator(::Val{4}, β) = @inbounds abs(β[1] +  3β[2] -   3β[3] -    β[4])
 @inline global_smoothness_indicator(::Val{5}, β) = @inbounds abs(β[1] +  2β[2] -   6β[3] +   2β[4] + β[5])
 @inline global_smoothness_indicator(::Val{6}, β) = @inbounds abs(β[1] + 36β[2] + 135β[3] - 135β[4] - 36β[5] - β[6])
 
-# Calculating Dynamic WENO Weights (wᵣ), either with JS weno, Z weno or VectorInvariant WENO
-for (side, coeff) in zip([:left, :right], (:Cl, :Cr))
-    biased_weno_weights = Symbol(side, :_biased_weno_weights)
-    biased_β = Symbol(side, :_biased_β)
-    
-    tangential_stencil_u = Symbol(:tangential_, side, :_stencil_u)
-    tangential_stencil_v = Symbol(:tangential_, side, :_stencil_v)
-
-    biased_stencil_z = Symbol(side, :_stencil_z)
-    
-    @eval begin
-        @inline function $biased_weno_weights(ψ, scheme::WENO{N, FT}, args...) where {N, FT}
-            @inbounds begin
-                β = beta_loop(scheme, ψ, $biased_β)
-                    
-                τ = global_smoothness_indicator(Val(N), β)
-                α = zweno_alpha_loop(scheme, β, τ, $coeff, FT)
-
-                return α ./ sum(α)
-            end
-        end
-
-        @inline function $biased_weno_weights(ijk, scheme::WENO{N, FT}, dir, ::VelocityStencil, u, v) where {N, FT}
-            @inbounds begin
-                i, j, k = ijk
-            
-                uₛ = $tangential_stencil_u(i, j, k, scheme, dir, u)
-                vₛ = $tangential_stencil_v(i, j, k, scheme, dir, v)
-                βᵤ = beta_loop(scheme, uₛ, $biased_β)
-                βᵥ = beta_loop(scheme, vₛ, $biased_β)
-
-                β  = beta_sum(scheme, βᵤ, βᵥ)
+# Calculating Dynamic WENO Weights (wᵣ), using the Z-WENO formulation
+@inline function biased_weno_weights(ψ, scheme::WENO{N, FT}, args...) where {N, FT}
+    β = beta_loop(scheme, ψ)
+                
+    τ = global_smoothness_indicator(Val(N), β)
+    α = zweno_alpha_loop(scheme, β, τ)
 
-                τ = global_smoothness_indicator(Val(N), β)
-                α = zweno_alpha_loop(scheme, β, τ, $coeff, FT)
+    return α ./ sum(α)
+end
 
-                return α ./ sum(α)
-            end
-        end
-    end
+@inline function biased_weno_weights(ijk, scheme::WENO{N, FT}, bias, dir, ::VelocityStencil, u, v) where {N, FT}
```

Comment:
here `bias` is a boolean where `true` means left biased and `false` means right biased? is that right? I think it makes sense to document this.